packages/block-library/src/cover/editor.scss

```diff
@@ -23,13 +22,14 @@
 	.wp-block-cover__placeholder-background-options {
 		width: 100%;
 	}
+}
 
-	// Apply max-width to floated items that have no intrinsic width.
-	[data-align="left"] &,
-	[data-align="right"] & {
-		max-width: $content-width / 2;
-		width: 100%;
-	}
+[data-align="left"] > .wp-block-cover,
+[data-align="right"] > .wp-block-cover,
+[data-align="left"] > .wp-block-cover-image,
+[data-align="right"] > .wp-block-cover-image {
+	max-width: $content-width / 2;
```

Comment:
Why is with being defined here?

Reply:
I'm just curious, it's not a comment to this PR.

Reply:
Just guessing

I believe if we don't do it and align a block (left or right) we won't notice any change as otherwise it will stay with a 100% width